Automattic/wp-calypso
# Jetpack Cloud Config Refinement

#### Changes proposed in this Pull Request

* Remove the following configs
   * `jetpack-cloud/backups`
   * `jetpack-cloud/scan`
   * `jetpack-cloud/scan-history`
   * `jetpack-cloud/settings`
* Rename the following configs
   * `jetpack-cloud/on-demand-scan` -> `jetpack/on-demand-scan`
   * `jetpack-cloud/backups-restore` -> `jetpack/backups-restore`
* Unifies config settings across most environments
 
#### Testing instructions

1. Verify Jetpack Cloud is unchanged
2. Verify Calypso is unchanged, still reveals Jetpack menu when adding `?flags=jetpack/features-section` to live/local url
